apps/src/templates/lessonOverview/StudentLessonOverview.jsx

```diff
@@ -12,20 +12,36 @@ import styleConstants from '@cdo/apps/styleConstants';
 import color from '@cdo/apps/util/color';
 import LessonNavigationDropdown from '@cdo/apps/templates/lessonOverview/LessonNavigationDropdown';
 import ResourceList from '@cdo/apps/templates/lessonOverview/ResourceList';
+import ProgressLessonContent from '@cdo/apps/templates/progress/ProgressLessonContent';
 import {studentLessonShape} from '@cdo/apps/templates/lessonOverview/lessonPlanShapes';
 import {linkWithQueryParams} from '@cdo/apps/utils';
 import Button from '@cdo/apps/templates/Button';
 import StyledCodeBlock from './StyledCodeBlock';
+import {levelWithProgressType} from '@cdo/apps/templates/progress/progressTypes';
+import {levelsForLessonId} from '@cdo/apps/code-studio/progressRedux';
 
 class StudentLessonOverview extends Component {
   static propTypes = {
     lesson: studentLessonShape.isRequired,
 
     // from redux
+    lessonLevels: PropTypes.arrayOf(levelWithProgressType),
     announcements: PropTypes.arrayOf(announcementShape),
     isSignedIn: PropTypes.bool.isRequired
   };
 
+  determineLevelDisplay = () => {
+    console.log(this.props.lessonLevels);
+    return this.props.lessonLevels ? (
```

Comment:
Empty arrays are truthy in javascript, I'd recommend checking length `this.props.lessonLevels.length` unless it can be null? In that case you'd probably want `this.props.lessonLevels?.length`

Reply:
Oh interesting... I think lessons without levels have null instead of an empty array, but I will switch to this anyway. Thanks!